Userland/Libraries/LibJS/Runtime/Intl/PluralRules.cpp

```diff
@@ -21,9 +21,9 @@ PluralRules::PluralRules(Object& prototype)
 ::Locale::PluralOperands get_operands(String const& string)
 {
     // 1.Let n be ! ToNumber(s).
-    char* end { nullptr };
-    auto number = strtod(string.characters(), &end);
-    VERIFY(!*end);
+    auto number_or_error = string.to_double(AK::TrimWhitespace::Yes);
```

Comment:
An `Optional<T>` is not an error - our usual naming for this is `maybe_foo`. But I think you can just skip the assertion and `release_value()`, should be clear from the comment above it